src/app/services/species.service.spec.ts

```diff
@@ -26,7 +26,7 @@ describe('SpeciesService', () => {
     httpClientSpy.get.and.returnValue(of(species));
     service.getAll().subscribe(data => {
```

Comment:
This test doesn't seem to be bug free so let's remove it. I also think you should approach it differently. Instead of checking for each species, which is unnecessary, you want to assert that the results match a certain set of expected data. That will give your test more consistency and simplify it.

Reply:
The test was removed, we are currently working on fixing that test.